src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java

```diff
@@ -0,0 +1,54 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+import org.bukkit.inventory.ItemStack;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ThrownExpBottleMockTest
+{
+
+	private ThrownExpBottleMock bottle;
+
+	@BeforeEach
+	void setUp()
+	{
+		ServerMock server = MockBukkit.mock();
+		bottle = new ThrownExpBottleMock(server, UUID.randomUUID());
+	}
+
+	@AfterEach
+	void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
+	@Test
+	void testGetItem()
+	{
+		assertEquals(Material.EXPERIENCE_BOTTLE, bottle.getItem().getType());
+	}
+
+	@Test
+	void testSetItem()
+	{
+		bottle.setItem(new ItemStack(Material.DIAMOND));
+
+		assertEquals(Material.DIAMOND, bottle.getItem().getType());
+	}
```

Comment:
Please also test that the amount is always one, no matter the used ItemStack's amount

Reply:
Done in the commit [b52e778](https://github.com/MockBukkit/MockBukkit/pull/913/commits/b52e77896fb4860405828817eb0a62da8f82a1ce)